src/Compilers/CSharp/Test/Semantic/Semantics/RawInterpolationTests_Handler.cs

```diff
@@ -11386,6 +11386,44 @@ .maxstack  4
 ");
     }
 
+    [Theory, WorkItem(59603, "https://github.com/dotnet/roslyn/issues/59603")]
+    [InlineData(@"$$$""""""{{1 + 2}}""""""")]
+    public void BracesNotEscaped(string expression)
```

Comment:
Consider also adding the two related scenarios that David reported:
1. four dollars and two curlies
2. four dollars and three curlies

Reply:
sure.  will do.